### PR TITLE
Fix for internal bug 1132014: Implement interface generates innacessible...

### PR DESCRIPTION
**Pull request by mavasani**
... attributes on implemented members

Fix is to remove inaccessible attributes from the generated members and its parameters.

### CODE REVIEW

**Comment by mavasani** [comment]
@tmeschter @srivatsn @jmarolf @shyamnamboodiripad @heejaechang @JohnHamby please review.

**Comment by srivatsn** [comment]
:+1:

File: src/Workspaces/Core/Portable/Shared/Extensions/IMethodSymbolExtensions.cs [hunk 1]
```diff
@@ -168,29 +168,39 @@ public static IMethodSymbol RenameParameters(this IMethodSymbol method, IList<st
...
         }
 
-        public static IMethodSymbol RemoveAttributeFromParametersAndReturnType(
-            this IMethodSymbol method, INamedTypeSymbol attributeType,
+        public static IMethodSymbol RemoveInaccessibleAttributesAndAttributesOfType(
+            this IMethodSymbol method, ISymbol accessibleWithin, INamedTypeSymbol removeAttributeType,
             IList<SyntaxNode> statements = null, IList<SyntaxNode> handlesExpressions = null)
         {
-            if (attributeType == null)
+            if (removeAttributeType == null)
```
Comment (JohnHamby) [inline src/Workspaces/Core/Portable/Shared/Extensions/IMethodSymbolExtensions.cs]: Is it possible for removeAttributeType to be null and there to also be inaccessible attributes that should be removed? If so, the inaccessible attributes won't be removed here. If not, consider adding a comment explaining why that situation cannot arise.

**Comment by JohnHamby** [comment]
I sign off.

File: src/Workspaces/Core/Portable/Shared/Extensions/IMethodSymbolExtensions.cs (hunk 1 above)
Comment (mavasani) [inline src/Workspaces/Core/Portable/Shared/Extensions/IMethodSymbolExtensions.cs]: Yes, you are right. I'll handle this case. Thanks.

**Comment by shyamnamboodiripad** [comment]
:+1:

**Pull request merged**